### DISCUSSION TITLE
[Bun] Add offical plugin for  bun

### DISCUSSION THREAD

**Discussion by jqhr** [solidjs/solid#1899] [Ideas]
[Bun](https://bun.sh/) is a very fast js runtime. And it support bundle and code transpile.
 Solidjs is the fastest jsx framework. Many developers can't wait to use Solidjs and Bun.
Solidjs has its own compiler, and not same as common jsx compiler.
Now, Bun has support [loader plugin](https://bun.sh/docs/runtime/plugins). To compile jsx directly.
Can the official support the development of a Bun plug-in?

**Comment by DaniGuardiola** [comment]
FYI, I built a plugin to compile Solid.js with Bun - https://github.com/DaniGuardiola/bun-plugin-solid
